uber/tests/email/test_automated_email_job.py

```diff
@@ -15,18 +15,13 @@ def send_all_emails_mock():
 
 @pytest.mark.usefixtures("email_subsystem_sane_setup")
 class TestSendAllAutomatedEmailsJob:
-    @pytest.mark.parametrize("c_at_the_con, c_send_emails, c_dev_box, expected_result", [
-        (True, True, True, 0),
-        (True, False, True, 0),
-        (True, True, False, 0),
-        (True, False, False, 0),
-        (False, True, True, 1),
-        (False, False, True, 1),
-        (False, True, False, 1),
-        (False, False, False, 0),
+    @pytest.mark.parametrize("c_send_emails, c_dev_box, expected_result", [
+        (True, True, 1),
+        (False, True, 1),
+        (True, False, 1),
+        (False, False, 0),
```

Comment:
Since we no longer bail out of running in on-site mode, it cut the number of permutations in half here.